lib/lib.js

```diff
@@ -31,12 +30,13 @@ const listDependencies = deps =>
     .join('\n')
 
 // injectDependencies :: Object -> String -> String
-const injectDependencies = (deps, code) => stripIndent`
-  // snippet dependencies
-  ${listDependencies(deps || [])}
+const injectDependencies = (deps, code) =>
+  `
+// snippet dependencies
+${listDependencies(deps || [])}
 
-  ${code}
-`
+${code}
+`.trim()
```

Comment:
PP: I like the `trim` but I don't like how indentation looks now, wdyt ?

Reply:
I don't like the indentation either. But let's say we indent it, and that the code is:

```
const answer = 42
console.log(answer)
```

If we indent it, the resulting snapshot will be:

```
  const answer = 42
console.log(answer)
```

It's not that big of a issue. You have way more experience with snapshots, what do you think about the trade-off here between prettier code and snapshots legibility?

Reply:
Yes, I would give up prettier code too. But can't we use `stripIndent` to achieve both?

Reply:
I think we can't, because the second line in `${code}` has no indentation, and `stripIndent` uses the less indented line. If we use `stripIndents` instead, we lose all the indentation in the snippet. Does that make sense?